libs/widget-lib/src/JsonRpcManager.ts

```diff
@@ -4,20 +4,36 @@ const JSON_PRC_V = '2.0'
 const TARGET_ORIGIN = '*'
 const EVENTS = ['connect', 'disconnect', 'close', 'chainChanged', 'accountsChanged']
 
+/**
+ * Manages JSON-RPC requests and interactions with an Ethereum provider.
+ */
 export class JsonRpcManager {
-  ethereumProvider: EthereumProvider | null = null
+  /** The Ethereum provider instance. */
```

Comment:
should we mention what null means? it means the widget would handle the web3 connection, right?

Reply:
Added a comment:
```
When is null the JSON-RPC manager is disconnected from the Ethereum provider
```